Story/NoobshireQuests.cs

```diff
@@ -0,0 +1,79 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class GiantTaleStoryWIP {
+  public IScriptInterface Bot =>IScriptInterface.Instance;
+  public CoreBots Core =>CoreBots.Instance;
+  public CoreStory Story = new();
+
+  public void ScriptMain(IScriptInterface bot) {
+    Core.SetOptions();
+    doAll();
+    Core.SetOptions(false);
+  }
+
+  public void doAll() {
+    Story.PreLoad(this);
+    RolithQuests();
+    LowellCatQuests();
+  }
+
+  public void RolithQuests() {
+    if (Core.isCompletedBefore(39)) return;
+
+    //Defend the Rats
```

Comment:
Add quest Ids as well next to quest name for better reachability while searching vs code